streams-bootstrap/build.gradle.kts

```diff
@@ -0,0 +1,46 @@
+description = "Collection of commonly used modules when writing a Kafka Streams Application"
+
+plugins {
+    id("com.github.davidmc24.gradle.plugin.avro") version "1.5.0"
+}
+
+dependencies {
+    val kafkaVersion: String by project
+    implementation(group = "org.apache.kafka", name = "kafka_2.13", version = kafkaVersion)
```

Comment:
@raminqaf mentioned this a while back: Why do we need this as dependency?

Reply:
I think it is because of StreamsResetter